dvc.yaml

```diff
@@ -0,0 +1,18 @@
+stages:
+  create_model:
+    cmd: python src/create_model.py
+    deps:
+    - data/train.tsv
+    - data/test.tsv
+    - data/validation.tsv
+    outs:
+    - output/model.joblib
```

Comment:
It outputs a few more files and this turned into a pickle file (and the name changed oops):
- `binarizer.pkl` 
- `classifier.pkl`
- `evaluation.json` 
- `vectorizer.pkl`